tests/test.py

```diff
@@ -91,9 +91,9 @@ def optional_test(testmethodorclass):
     all optional packages have been installed.
     """
 
-    return unittest.skipIf(not os.environ.get(OPTIONAL_VAR), "optional test")(
-        testmethodorclass
-    )
+    return unittest.skipIf(
```

Comment:
I don't believe this fixes this. Like don't you want it to evaluate to `False` if you set the env variable to 0 or False or something? 
```
(trulens_3_11) dkurokawa@KJ6R9J3J4N trulens % cat cow_testing.py                 
import os

if os.environ.get("COW_ENV_VAR", False):
    print("True")
else:
    print("False")
(trulens_3_11) dkurokawa@KJ6R9J3J4N trulens % unset COW_ENV_VAR                 
(trulens_3_11) dkurokawa@KJ6R9J3J4N trulens % python cow_testing.py 
False
(trulens_3_11) dkurokawa@KJ6R9J3J4N trulens % COW_ENV_VAR=False python cow_testing.py
True
(trulens_3_11) dkurokawa@KJ6R9J3J4N trulens % COW_ENV_VAR=0 python cow_testing.py    
True
```

Reply:
It is using python's view of true and I believe any non-empty string is true. Changed it to cast to bool first to further reduce confusion.

Reply:
* changed to a test for several specific strings